src/app/reports/run-report/table-and-sms/table-and-sms.component.ts

```diff
@@ -80,7 +80,22 @@ export class TableAndSmsComponent implements OnChanges {
    */
   downloadCSV() {
     const headers = this.displayedColumns;
-    let csv = this.csvData.map((object: any) => object.row.join());
+    headers.forEach((header: string|number) => {
+      header = header.toString();
+      if ((header.indexOf(',')) !== -1) {
+        header = `\'${header}\'`;
+      }
+    });
+    let csv = this.csvData.map((object: any) => {
+      object.row = object.row.map((rowEl: string|number) => `\'${rowEl}\'`);
+      object.row.forEach((rowEl: string|number) => {
+        rowEl = rowEl.toString();
+        if ((rowEl.indexOf(',')) !== -1) {
+          rowEl = `\'${rowEl}\'`;
+        }
+      });
```

Comment:
Wouldn't replacing commas for each `object.row` with space (with .replace()), suffice?

Reply:
I did it to wrap up only those which has , in it. Doing it for object.row would cause it to be there for all of them. Should I update it for all row?

Reply:
I guess .replace would have no effect if there is no comma, still check if it works otherwise this is fine as well.

Reply:
-> a=['Taushar Kaushik,Kaushik Tushar']
[ 'Taushar Kaushik,Kaushik Tushar' ]
-> a[0].replace(',',' ')
'Taushar Kaushik Kaushik Tushar'
this kind of string might cause confusion for the end-user. What do you think? Just a confirmation and I would update the pr.

Reply:
You can specify the replaced string to be '\' or '-' instead of space.

Reply:
Resolved, please take a look when you are free. Thanks in advance.